wallet/common/symbol/src/utils/transaction-from-symbol.js

```diff
@@ -0,0 +1,603 @@
+import { addressFromPublicKey } from './account';
```

Comment:
These are from Symbol-sdk to Wallet objects?

Reply:
Yes they are.